lib/twilio.rb

```diff
@@ -0,0 +1,20 @@
+require 'rubygems'
+require 'twilio-ruby'
+
+class Sendsms
+
+  def send_message(body)
+
+    account_sid = 'AC69cef69991d1c8e42e3bfff281157e71'
```

Comment:
hide your credentials with environmental variables 
- https://www.twilio.com/blog/2017/01/how-to-set-environment-variables.html
- https://github.com/bkeepers/dotenv